central/productusage/csv/csv_handler.go

```diff
@@ -0,0 +1,97 @@
+package usagecsv
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/gogo/protobuf/types"
+	"github.com/pkg/errors"
+	datastore "github.com/stackrox/rox/central/productusage/datastore/securedunits"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/csv"
+	"github.com/stackrox/rox/pkg/errox"
+	grpcErrors "github.com/stackrox/rox/pkg/grpc/errors"
+	"github.com/stackrox/rox/pkg/protoconv"
+)
+
+var (
+	zeroTime  = time.Time{}
+	csvHeader = csv.Row{"Timestamp", "Nodes", "CPU Units"}
+)
+
+func getSecuredUnitsConverter() csv.Converter[storage.SecuredUnits] {
+	record := make(csv.Row, 3)
+	return func(m *storage.SecuredUnits) csv.Row {
+		record[0] = ""
+		if m.GetTimestamp() != nil {
+			if t, err := types.TimestampFromProto(m.GetTimestamp()); err == nil {
+				record[0] = t.UTC().Format(time.RFC3339)
+			}
+		}
+		record[1] = fmt.Sprint(m.GetNumNodes())
+		record[2] = fmt.Sprint(m.GetNumCpuUnits())
+		return record
+	}
+}
+
+// CSVHandler returns an HTTP handler function that serves product usage data as CSV.
+// The handler accepts two optional string parameters "from" and "to", which are
+// expected to represent a timestamp in RFC3339 (or 'ISO') format.
+// The parameters define a time range in which the data is searched. The result
+// includes data not older than "from" and older than "to", i.e., [from, to).
+// If "from" is ommited, an 0 timestamp is taken instead. If "to" is ommited,
+// the current time is taken.
```

Comment:
For the future, it's _always_ good to leave an example, something like:
```
<Central URL>/api/usage/csv?from=2023-07-31&to=2023-07-31'
```

Reply:
Will do in a follow-up. Will also correct "an 0" to "a 0".